group.py

```diff
@@ -1,5 +1,19 @@
 """An example of how to represent a group of acquaintances in Python."""
 
 # Your code to go here...
+week3_group=[{
```

Comment:
Should be named `my_group` -
```suggestion
my_group=[{
```